integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -742,17 +759,22 @@ protected String getExpectedTableData(SparkTableInfo table) {
   }
 
   protected String getCreateSimpleTableString(String tableName) {
-    return getCreateSimpleTableString(tableName, false);
+    return getCreateSimpleTableString(tableName, false, false);
   }
 
-  protected String getCreateSimpleTableString(String tableName, boolean isExternal) {
```

Comment:
I think there is no need to add location in `getCreateSimpleTableString`,  we could append `Location xxx` to  `getCreateSimpleTableString`  like partition info in  `testCreateHiveFormatPartitionTable`.